src/plugins/mer/merplugin.cpp

```diff
@@ -168,6 +174,57 @@ bool MerPlugin::initialize(const QStringList &arguments, QString *errorString)
             MerQmlLiveBenchManager::startBench);
     menu->addAction(startQmlLiveBenchCommand);
 
+    Utils::MacroExpander *expander = Utils::globalMacroExpander();
+    expander->registerVariable("mer:merssh", tr("Location of merssh"),
+                               []()
+    {
+        return QString(Core::ICore::libexecPath() + QLatin1String("/merssh") + QStringLiteral(QTC_HOST_EXE_SUFFIX));
+    });
```

Comment:
Add MER_WRAPPER_LUPDATE to wrapperScripts in mertarget.cpp, then you can specify the executable as %{CurrentBuild:Env:MER_SSH_SDK_TOOLS}/lupdate

Reply:
That still feels like just a more complicated way of doing it. Adds complexity without added benefit.

Reply:
Looking at the scripts (qmake etc), their only function is to add MER_SSH_TARGET_NAME and MER_SSH_SDK_TOOLS to envinroment before calling merssh. Neither of those variables are needed in this case.

Reply:
There is a pattern. The pattern is used consistently in 100% cases even if there are cases when the pattern adds no functional value. The value comes from using a pattern consistently.

Moreover if I am not wrong in this particular case the pattern enables adding just one line of code instead of six lines. But even if this was not true, the formerly said applies - there is a pattern, so let's follow it.

Reply:
Now using the script